R/db.R

```diff
@@ -37,13 +37,13 @@ st_read_db = function(conn = NULL, table, query = paste("select * from ", table,
 			SRID = dbGetQuery(conn, paste0("select srid from geometry_columns where f_table_name = '", table, "';"))[[1]]
 			ret = dbGetQuery(conn, paste("select proj4text from spatial_ref_sys where srid =", SRID, ";"))
 			if (nrow(ret))
-				ret[[1]]
+				SRID
```

Comment:
Ah, I wasn't aware of that and thought the EPSG would always be properly resolved.

I went this route because it would give me a complete "crs" with both EPSG and proj4string.
Would the newly committed version with constructing a complete "crs" beforehand be better then?

On the other hand, writing to the DB is currently only based on the SRID. Should this also be based on the proj4string instead then?